packages/components/src/components/Table/TableCell.tsx

```diff
@@ -19,6 +21,8 @@ type Padding = {
     bottom?: SpacingValues;
     left?: SpacingValues;
     right?: SpacingValues;
+    vertical?: SpacingValues;
+    horizontal?: SpacingValues;
```

Comment:
I like this one. 

Btw IMHO we should use it also for `margin` in `frameProps`. And maybe also adding this one:
`padding={spacings.sm}` (and the same for `margin`)

Reply:
I've already added it to margin in frameProps:)